cms/templates/textbooks.html

```diff
@@ -54,7 +59,9 @@ <h3 class="sr">${_("Page Actions")}</h3>
 
   <div class="wrapper-content wrapper">
     <section class="content">
-      <article class="content-primary" role="main"></article>
+      <article class="content-primary" role="main">
+        <div id="no-textbooks-container"></div>
```

Comment:
Placeholder div. `WrappedNoTextbooks` will get rendered into here as per the `id` argument to the `renderReact` call above.